tests/apps/core/test_cache.py

```diff
@@ -0,0 +1,409 @@
+"""
+Test suite for the cache module in the `core` application
+
+Credits :
+ - https://github.com/lamoda/django-sentinel
```

Comment:
I am concerned that there is no license associated with this project. It has been removed while undeprecating the original project: https://github.com/lamoda/django-sentinel/commit/31d0d7d40f016a16b8995fa01eb85080ae7f6a09

Reply:
I don't know why the LICENSE file has been removed, but I don't think it's a problem because :

- The original project code is under MIT licence 
- The new python package (`django-sentinel`) is referenced on PyPi with MIT license (see [here](https://pypi.org/project/django-sentinel/) and [here](https://github.com/lamoda/django-sentinel/blob/673696d0c5826bdca299662f1d22403d08476321/setup.py#L35))

Reply:
As stated by the `setup.cfg` file, we can consider the forked project license as still MIT-based.